javascript/modules/client/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@aragon/sdk-client",
   "author": "Aragon Association",
-  "version": "0.2.0-alpha",
+  "version": "0.3.0-alpha",
   "license": "MIT",
   "main": "dist/index.js",
```

Comment:
Shouldn't we also update common to 0.2.1-alpha ?

Reply:
Yes, I was following semantic versioning, and I would not consider this a bug fix, that's the explanation behind the `0.3.0` instead of the `0.2.1`